### PR TITLE
Redesigned mobile nav

### PR DESCRIPTION
**Pull request by musidlo**
image

### CODE REVIEW

File: packages/app/components/NavSheet.tsx [hunk 1]
```diff
@@ -37,7 +38,8 @@ export function NavSheet({
     >
       <Sheet.Overlay />
       <Sheet.Frame zIndex={1} height={isWeb ? '100vh' : '100%'} pt={isPwa && sat}>
-        <Stack w="100%" ai="flex-end" jc="flex-end" px="$6" pt="$6" pb="$2">
+        <XStack w="100%" ai="center" jc="space-between" px="$4" pt="$6" pb="$2">
+          <Paragraph size={'$10'}>/send</Paragraph>
```
Comment (youngkidwarrior) [inline packages/app/components/NavSheet.tsx:42]: This doesn't match the logo styling in the rest of the app.

Let's use the svg instead
```suggestion
       <Link href="/send">
                      <IconSendLogo size={'$2.5'} color={'$color12'} />
                    </Link>
```

**Review by youngkidwarrior — CHANGES_REQUESTED** [review]
One change

Using `first` and `last` works for now, but look into [Tamagui UI Groups](https://tamagui.dev/ui/group) \

**Comment by musidlo** [comment]
> One change
> 
> Using `first` and `last` works for now, but look into [Tamagui UI Groups](https://tamagui.dev/ui/group) \

@youngkidwarrior those links are put placed in scroll view. If I put there a group it doesnt make sense to keep in in scroll view imo. It's either scroll view with individual items or buttons group. originally it was scroll view so I wanted to keep it this way. let me know what you think

File: packages/app/components/sidebar/SideBarNavLink.tsx [hunk 2]
```diff
@@ -33,6 +31,8 @@ export function SideBarNavLink({
         gap="$3"
         display="flex"
         alignItems="center"
+        {...props}
+        href={props.disabled ? '' : props.href}
```
[the remark this answers is not in the document]
Comment (musidlo) [reply packages/app/components/sidebar/SideBarNavLink.tsx:35]: that's how I wanted it to be

**Comment by youngkidwarrior** [comment]
leave it for now. Better to reduce complexity. Leave it a lil smelly lol

**Review by youngkidwarrior — APPROVED** [review]

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Dec 17, 12:40 AM EST**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/946).

**Pull request merged**